src/include/wiredtiger.in

```diff
@@ -261,6 +264,9 @@ struct __wt_cursor {
 	 * @param cursor the cursor handle
 	 * @param ... pointers to hold value fields corresponding to
 	 * WT_CURSOR::value_format.
+	 * The API don't have a way to validate the argument types passed in,
```

Comment:
Better to frame sentence as "will not validate the argument types passed in" rather than "don't have a way to validate"

Reply:
corrected.